src/SearchableEntity.php

```diff
@@ -62,7 +62,6 @@ public function getIndexUid(): string
     public function getSearchableArray(): array
     {
         $context = [
-            /* @phpstan-ignore-next-line */
```

Comment:
Hello @norkunas thanks for your contribution!

I could be definitely wrong about this, but I think what @curquiza wants when she created the #158 issue, was to fix the problem not by modifying the place where we put the errors. 
Because in this case we still are not ok with the phpstan standards.

I'm not sure how to fix the issue, but I'm not sure if this fixes the issue as well!

Reply:
Oh sorry @norkunas I saw your comment in the description!

> Maybe this will make you a little bit happier than having ignore annotation in code until there will be a proper fix for this :)

Reply:
I like the `phpstan-baseline.neon` way to handle the TODO's of the project, but I'm not sure if is worth adding. Can you help me understand the value behind this addition? 

Thank you in advance! You rocks! 🥇

Reply:
Hi, it's easier to found in one place warnings that should be fixed someday,but in this case it's just a single warning 😅 we can leave it as is, but it'd be easier for others to discover this :)

Reply:
Ok, good for me! 🍣